bundles/org.openhab.binding.modbus/src/main/resources/OH-INF/thing/bridge-tcp.xml

```diff
@@ -58,6 +58,13 @@
 				<default>1</default>
 				<advanced>true</advanced>
 			</parameter>
+			<parameter name="afterConnectionDelayMillis" type="integer" min="0" unit="ms">
```

Comment:
Please update the file modbus.properties file to include the 2 new parameters.
This can be done automatically by running this command: `mvn org.openhab.core.tools:i18n-maven-plugin:3.2.0:generate-default-translations`